### ISSUE TITLE
include_target=True in result.to_dataset() causes crash

### ISSUE THREAD

**Issue by simonmicro** [flemmerich/pysubgroup#40]
This is a bug. When I call...
```
result.to_dataframe(include_target=True)
```
...this happens:
```
Traceback (most recent call last):
  File "...train_and_evaluate_models.py", line 390, in <module>
    result.to_dataframe(include_target=True).to_csv(os.path.join(explainGraphDir, 'subgroups.csv'))
  File ".../site-packages/pysubgroup/utils.py", line 234, in to_dataframe
    df = pd.DataFrame(res, columns=headers, dtype=np.float64)
  File ".../site-packages/pandas/core/frame.py", line 745, in __init__
    arrays, columns, index = nested_data_to_arrays(
  File ".../site-packages/pandas/core/internals/construction.py", line 510, in nested_data_to_arrays
    arrays, columns = to_arrays(data, columns, dtype=dtype)
  File "/.../site-packages/pandas/core/internals/construction.py", line 875, in to_arrays
    content, columns = _finalize_columns_and_data(arr, columns, dtype)
  File ".../site-packages/pandas/core/internals/construction.py", line 972, in _finalize_columns_and_data
    raise ValueError(err) from err
ValueError: 16 columns passed, passed data had 17 columns
```

Turns out https://github.com/flemmerich/pysubgroup/blob/be8dc38eec65cf57a0bb069c08753482acc695af/pysubgroup/utils.py#L228 does not care to extend the tables header to include the column for the targets name, which causes pandas to crash upon parsing the table structure.

**Comment by langenberg** [comment]
Hey there!
I get a similar error message. Downgrading pandas below version 2 (1.1.5 in my case) fixed the issue for me.
@flemmerich Can you have a look into this? Lots of thanks!
Best,
Benedikt

**Comment by mgbckr** [comment]
Thanks for reporting. This should be fixed with #42

**Comment by mgbckr** [comment]
#42 is merged and released (version `0.7.6`) and should fix the issue. Please let us know if we missed something. Closing for now.
